joblib/externals/copy_dill.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Do a local install of dill.
+
+pip install --target=. dill
```

Comment:
Nice to see that you are not using easy_install!

Can you make this script executable via chmod?

Also I believe you need to rm -rf dill in order to install the latests dill if the dill folder already exists.